### ISSUE TITLE
Pylint missing attributes error

### ISSUE THREAD

**Issue by ssd352** [JetBrains/teamcity-messages#215]
When I run PyLint on TeamCity, I get the following error:
```
Attributes id, name, category, description are required to 'inspectionType' service message
```
I checked the output of `pylint --output-format=teamcity.pylint_reporter.TeamCityReporter  *.py` and apparently it is missing a mandatory `description` attribute.
```
##teamcity[inspectionType timestamp='<some-timestamp>' category='warning' id='W0511' name='fixme']
```

**Comment by dh0d** [comment]
I have the same issue. Version 2018.2.4 (build 61678)

**Comment by PetrWolf** [comment]
Hi! I think this is related to a recent change in PyLint, in which the way to get message description changed. 

Please can you check what version of PyLint you're using?

I will take a look

**Comment by ssd352** [comment]
I am currently using Pylint 2.3.1. What was the latest version of Pylint that used to work?

**Comment by PetrWolf** [comment]
It should work up to an including 2.2.2. The incompatibility is with 2.3 and above. I have [contacted](https://github.com/PyCQA/pylint/issues/2936) the PyLint team for suggestions and will try to make a fix soon.

**Comment by PetrWolf** [comment]
I have created PR #217 based on my best understanding of the PyLint interface for now. This should resolve the issue and make the code and tests more robust going forward.

If the PyLint maintainers suggest a better option in the future, I will come back and implement it.

**Comment by PetrWolf** [comment]
Thanks for merging #217! 

Do you think this issue can be closed now? 
And when do you plan the next release?

**Comment by PetrWolf** [comment]
Is a new release planned to include a fix to this issue? Thanks!

**Comment by RemySphere** [comment]
Hello,
We're experiencing the same issue here; going back to PyLint 2.2 with Astroid 2.0.0 is ok for now but we're also awaiting a release!

Regards

**Comment by shalupov** [comment]
Finally released 1.26, sorry for the wait

**Comment by aviceconti** [comment]
Hi there,
I had the same problem and although I've installed the 1.26 the problem still persitsts.
It seems that at line 26 the attribute name is still 'descr' and not description.

>         return " ".join(definition.descr for definition in linter.msgs_store.get_message_definitions(msgid))

**Comment by shalupov** [comment]
@aviceconti please tell me *exact* version of pylint and python

**Comment by aviceconti** [comment]
Hi there,
thanks for the quick response
pytest-pylint 2.3.1
python 3.7.4

Cheers

**Comment by shalupov** [comment]
@aviceconti released 1.27, please try again
